JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -190,16 +190,18 @@ module Microsoft.ApplicationInsights {
             if (!Telemetry.PageViewPerformance.isPerformanceTimingSupported()) {
                 // no navigation timing (IE 8, iOS Safari 8.4, Opera Mini 8 - see http://caniuse.com/#feat=nav-timing)
                 _InternalLogging.throwInternalNonUserActionable(LoggingSeverity.CRITICAL,
```

Comment:
Even though it's a known issue - it's still unknown what the percentage will be in, say, one month. Currently it's affecting 1% of customers, so we dicided to send 0 duration. If it becomes 10% (more safari oriented applications, or browser demography changes) we'll have to do something about it. Without this telemetry we'll go blind